src/dso_api/dynamic_api/serializers.py

```diff
@@ -300,14 +310,22 @@ class DynamicBodySerializer(DynamicSerializer):
 
     def get_fields(self):
         fields = super().get_fields()
+        ds = self.Meta.model.get_dataset()
+        identifier_fields = [getattr(ds, "identifier")]
+        temporal = getattr(ds, "temporal")
+        if temporal and "identifier" in temporal:
```

Comment:
1. `getattr(ds, "temporal")` zal altijd een AttributeError geven als deze ontbreekt, dus `ds.temporal` is hier voldoende. Tenzij je `getattr(ds, "temporal", None)` bedoeld.
2. Ontbreekt `temporal["identifier"]` soms? En zo ja, waarom?

Reply:
1. ook hier is de `getattr` overbodig.
2. Als de dataset geen temporele configuratie heeft, geeft temporal een `None` terug. Dit komt vaak voor en is 'by design'.
3. Op 1 regel lastig omdat je temporal als property maar 1x wilt ophalen.